Repository/NuGetSearchContext.cs

```diff
@@ -0,0 +1,108 @@
+﻿//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//  http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+//
+
+namespace Microsoft.PackageManagement.NuGetProvider
+{
+    using Microsoft.PackageManagement.Provider.Utility;
+    using System.Collections.Generic;
+
+    /// <summary>
+    /// Contains all required search information.
+    /// </summary>
+    public class NuGetSearchContext
```

Comment:
if we can make 
class, method internal, please make them all internal.

Reply:
Must be public

---
In reply to: [154461227](https://github.com/OneGet/NuGetProvider/pull/25#discussion_r154461227) [](ancestors = 154461227)